### PR TITLE
Add precision on CRS selector and OTF reprojection

### PR DESCRIPTION
**Pull request by DelazJ**
Insert a section to be reached from QGIS application Help button
Fix description
Add tip and note on the behavior of the CRS selection tool (inspired a bit by recent discussion on dev ML - actually a frequent confusion for new users)

### CODE REVIEW

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 1]
```diff
@@ -447,8 +447,18 @@ Map View
 
 Also called **Map canvas**, this is the "business end" of QGIS ---
 maps are displayed in this area. The map displayed in this window
-will depend on the vector and raster layers you have chosen to load
-(see sections that follow for more information on how to load layers).
+will depend on the vector and raster layers you have chosen to load.
+
+When you add a layer (see e.g. :ref:`vector_loading_file`), 
+QGIS automatically looks for the Coordinate Reference System (CRS) and zooms to
+the layer extent if you work in a blank QGIS project. If there are already files
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/qgis_gui.rst]: already layers?

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 2]
```diff
@@ -447,8 +447,18 @@ Map View
 
 Also called **Map canvas**, this is the "business end" of QGIS ---
 maps are displayed in this area. The map displayed in this window
-will depend on the vector and raster layers you have chosen to load
-(see sections that follow for more information on how to load layers).
+will depend on the vector and raster layers you have chosen to load.
+
+When you add a layer (see e.g. :ref:`vector_loading_file`), 
+QGIS automatically looks for the Coordinate Reference System (CRS) and zooms to
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/qgis_gui.rst]: looks for its CRS

File: source/docs/user_manual/introduction/qgis_gui.rst (hunk 1 above)
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/qgis_gui.rst]: would be nice to mention that the layer's CRS will be used for the project.

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 3]
```diff
@@ -447,8 +447,18 @@ Map View
...
+the layer extent if you work in a blank QGIS project. If there are already files
+in your project, the file will just be added, and in the case it has features
+falling in the map canvas extent, they will be visualized. If the file is in
+another CRS, you must :guilabel:`Enable on-the-fly CRS transformation` from the
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/qgis_gui.rst]: another CRS --> different CRS from the project

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 4]
```diff
@@ -447,8 +447,18 @@ Map View
...
+:menuselection:`Project --> Project Properties --> CRS` (see :ref:`otf_reprojection`).
+The newly added layer should now be visible if data are available in the current
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/qgis_gui.rst]: this part seems repeated

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 5]
```diff
@@ -93,41 +93,42 @@ of the raster and vector properties dialog (see :ref:`label_generaltab` for
...
+
+There are three ways to enable On The Fly Reprojection:
+
+* Select |projectProperties| :menuselection:`Project Properties --> CRS` from the
+  :menuselection:`Project` ( or |kde| :menuselection:`Settings`) menu. You can then
+  activate the |checkbox| :guilabel:`Enable on the fly CRS transformation` checkbox
+  in the |crs| :guilabel:`CRS` tab and select the CRS to use (see :ref:`crs_selector`)
+* Click on the |geographic| :sup:`CRS status` icon in the lower right-hand
+  corner of the status bar, leading you to the previous dialog.
+* Turn OTF on by default in the |crs| :guilabel:`CRS` tab of the
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]: I would move this "way" of turning OTF on to when you say that the default behaviour is having it off.

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 6]
```diff
@@ -59,9 +59,12 @@ modified), and the number of columns, rows and no-data values of the raster.
 Coordinate reference system
 ...........................
 
-Here, you find the coordinate reference system (CRS) information printed as a
-PROJ.4 string. If this setting is not correct, it can be modified by clicking
-the **[Specify]** button.
+Display the current coordinate reference system of the layer as a PROJ.4 string. You
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]: Displays?

Comment (SrNetoChan) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]: Displays the layer's Coordinate Reference System (CRS) as PROJ.4 string.

File: source/docs/user_manual/working_with_raster/raster_properties.rst [hunk 7]
```diff
@@ -59,9 +59,12 @@ modified), and the number of columns, rows and no-data values of the raster.
...
+can change the projection, selecting a recently used one in the drop-down list or
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_raster/raster_properties.rst]: you can change the layer's CRS,

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 8]
```diff
@@ -44,20 +44,23 @@ There are several options available:
...
+* Display the :guilabel:`Layer source` of the vector layer
 * Define the :guilabel:`Data source encoding` to define
   provider-specific options and to be able to read the file
 
 Coordinate Reference System
 ---------------------------
 
-* :guilabel:`Specify` the coordinate reference system. Here, you
-  can view or change the projection of the specific vector layer.
+* Display the current coordinate reference system of the layer as a PROJ.4 string. You
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: Displays the layer's Coordinate Reference System as...

File: source/docs/user_manual/working_with_vector/vector_properties.rst [hunk 9]
```diff
@@ -44,20 +44,23 @@ There are several options available:
...
+  can change the projection, selecting a recently used one in the drop-down list or
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_vector/vector_properties.rst]: change the CRS

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 10]
```diff
@@ -447,8 +447,18 @@ Map View
...
 maps are displayed in this area. The map displayed in this window
-will depend on the vector and raster layers you have chosen to load
-(see sections that follow for more information on how to load layers).
+will depend on the vector and raster layers you have chosen to load.
+
+When you add a layer (see e.g. :ref:`vector_loading_file`), 
+QGIS automatically looks for the Coordinate Reference System (CRS) and zooms to
+the layer extent if you work in a blank QGIS project. If there are already files
+in your project, the file will just be added, and in the case it has features
+falling in the map canvas extent, they will be visualized. If the file is in
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/qgis_gui.rst]: in the current map canvas extent.

If the layer is in

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 11]
```diff
@@ -447,8 +447,18 @@ Map View
...
 Also called **Map canvas**, this is the "business end" of QGIS ---
 maps are displayed in this area. The map displayed in this window
-will depend on the vector and raster layers you have chosen to load
-(see sections that follow for more information on how to load layers).
+will depend on the vector and raster layers you have chosen to load.
+
+When you add a layer (see e.g. :ref:`vector_loading_file`), 
+QGIS automatically looks for the Coordinate Reference System (CRS) and zooms to
+the layer extent if you work in a blank QGIS project. If there are already files
+in your project, the file will just be added, and in the case it has features
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/qgis_gui.rst]: the layer will just load

File: source/docs/user_manual/introduction/qgis_gui.rst [hunk 12]
```diff
@@ -447,8 +447,18 @@ Map View
 
 Also called **Map canvas**, this is the "business end" of QGIS ---
 maps are displayed in this area. The map displayed in this window
```
Comment (SrNetoChan) [inline source/docs/user_manual/introduction/qgis_gui.rst]: Not related to this PR, would be nice to improve this sentence. For starters, I would remove any reference to "Map view". This is probably the only place where someone calls it that. So I would assume the "Map canvas" as the official name, IMHO.

Then, I would say that the map canvas dos not display maps, it displays the load layers's features/geometries.

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 13]
```diff
@@ -93,41 +93,42 @@ of the raster and vector properties dialog (see :ref:`label_generaltab` for
...
    provides two CRS shortcuts. :guilabel:`Set layer CRS` takes you directly
    to the Coordinate Reference System Selector dialog (see figure_projection_project_).
    :guilabel:`Set project CRS from Layer` redefines the project CRS using
    the layer's CRS.
 
-.. index:: On-the-fly reprojection
+.. index:: CRS; On-the-fly reprojection
+.. _otf_reprojection:
 
 Define On The Fly (OTF) Reprojection
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]: This is another misleading title, we always call it "On-The-Fly Transformation", we should either change the GUI (which makes no sense now that the OFT was removed from QGIS 3.0), or change the title to match the GUI

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 14]
```diff
@@ -93,41 +93,42 @@ of the raster and vector properties dialog (see :ref:`label_generaltab` for
...
-   :menuselection:`Project` (Gnome, macOS) or :menuselection:`Settings` (KDE,
-   Windows) menu.
-#. Click on the |geographic| :sup:`CRS status` icon in the lower right-hand
-   corner of the status bar.
-#. Turn OTF on by default in the :guilabel:`CRS` tab of the
-   :guilabel:`Options` dialog by selecting |checkbox|
-   :guilabel:`Enable 'on the fly' reprojection by default` or :guilabel:`Automatically
-   enable 'on the fly' reprojection if layers have different CRS`.
+not activated by default. When OTF is off, each layer is drawn using the
+coordinates as read from the data source. When on, the coordinates in each
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]: original coordinates

File: source/docs/user_manual/working_with_projections/working_with_projections.rst [hunk 15]
```diff
@@ -93,41 +93,42 @@ of the raster and vector properties dialog (see :ref:`label_generaltab` for
...
    the layer's CRS.
 
-.. index:: On-the-fly reprojection
+.. index:: CRS; On-the-fly reprojection
+.. _otf_reprojection:
 
 Define On The Fly (OTF) Reprojection
 ====================================
 
 QGIS supports OTF reprojection for both raster and vector data. However, OTF is
```
Comment (SrNetoChan) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]: QGIS supports On-The-Fly (OTF) reprojection for both raster and vector data. However, OTF is

**Comment by SrNetoChan** [comment]
@DelazJ The  PR Looks good to merge, most of the comments are my opinion, you can just ignore them. :-) 

Thanks for working on this.

**Comment by DelazJ** [comment]
@SrNetoChan Thanks for your precise review. I like that.
At first sight, except some comments that might need discussion, your points are most of the time valid. I'll make changes when I have time.

File: source/docs/user_manual/working_with_projections/working_with_projections.rst (hunk 13 above)
Comment (DelazJ) [inline source/docs/user_manual/working_with_projections/working_with_projections.rst]: actually, the expression "otf reprojection" is used in global settings crs dialog.
About 3.0 changes, i'd say let's fix 2.18 doc yet.

**Pull request merged**